btrace-core/src/test/java/org/openjdk/btrace/core/BTraceUtilsTest.java

```diff
@@ -0,0 +1,30 @@
+package org.openjdk.btrace.core;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ReflectiveFieldAccessTest {
```

Comment:
The last thing - please, rename also the file such that it corresponds to the Java convention of having same named files and classes.